### DISCUSSION TITLE
Running preprocessor separately

### DISCUSSION THREAD

**Discussion by Banbury** [Mikubill/sd-webui-controlnet#422] [General]
Running ControlNet without a preprocessor works fine form me. But using a preprocessor slows down image generation to a crawl. 
Is there a way to run a preprocessor on its own?
This would be useful for batch generation of control images, too.

**Comment by martinobettucci** [comment]
Just click on preview, download the result and post it on the controlnet canvas. Disable the préprocess. Enjoy large batch generation 😉
